etc/inc/allow-common-devel.inc

```diff
@@ -38,3 +38,10 @@ noblacklist ${HOME}/.bundle
 # Rust
 noblacklist ${HOME}/.cargo
 noblacklist ${HOME}/.rustup
+
+#Arduino
```

Comment:
Code looks fine. We do `sort` these groups `alphabetically`. Please move it accordingly.

Reply:
Thanks for the hint!